src/main/java/org/primefaces/util/CalendarUtils.java

```diff
@@ -245,7 +245,6 @@ public static final String getValue(FacesContext context, UICalendar calendar, O
                     javax.faces.convert.DateTimeConverter nativeConverter = (javax.faces.convert.DateTimeConverter) converter;
                     if (PrimeApplicationContext.getCurrentInstance(context).getEnvironment().isAtLeastJsf23()) {
                         Field field = javax.faces.convert.DateTimeConverter.class.getDeclaredField("type");
-                        field.setAccessible(true);
```

Comment:
Sonar recommended removing this accessible set and i verified it was having no effect.